apps/website/package.json

```diff
@@ -17,7 +17,7 @@
     "react-is": "^18.2.0",
     "react-syntax-highlighter": "^15.5.0",
     "remark-remove-comments": "^0.2.0",
-    "rooks": "6.2.0"
```

Comment:
I wonder if we can somehow get best of both worlds but not sure how exactly with changesetscli

Reply:
Ohh, I see.

Then I'm curious how do you test rooks during development?

Reply:
@imbhargav5 is the rooks in devDependencies also added by changesets?

Reply:
@qqpann At this point of time, I only just test it with test cases. I don't really test a hook with the web app, although I see it can be useful. Mainly , my intention is to keep the webapp solely run with sandpack so that we have good demos to show that are spawned up in sandpack alongside the codesnippet. For local testing we can figure out something else, like perhaps a folder inside rooks called playground and just test it there. there are probably better ideas than this, but I am just thinking of a quick win.

Reply:
Thanks for sharing your intentions! Sounds good.
A playground in rooks or under apps sounds good to me.

I just reverted my changes in package.json for now.